### PR TITLE
Improved UX for Nearby Export

### PR DESCRIPTION
**Pull request by kanahia1**
**Description (required)**

What changes did you make and why?

**Tests performed (required)**

Tested prodDebug on Samsung S21 FE with API level 33.

**Screenshots (for UI changes only)**

https://github.com/commons-app/apps-android-commons/assets/114223204/c8c5636b-1874-46de-8a9f-7ee24c4ac289

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java [hunk 1]
```diff
@@ -396,14 +406,17 @@ public boolean onMenuItemClick(@NonNull MenuItem item) {
...
+                        .fromPixels(mapView.getWidth(), 0);
+                    IGeoPoint screenBottomLeft = mapView.getProjection()
+                        .fromPixels(0, mapView.getHeight());
                     fr.free.nrw.commons.location.LatLng screenTopRightLatLng = new fr.free.nrw.commons.location.LatLng(
                         screenBottomLeft.getLatitude(), screenBottomLeft.getLongitude(), 0);
                     fr.free.nrw.commons.location.LatLng screenBottomLeftLatLng = new fr.free.nrw.commons.location.LatLng(
                         screenTopRight.getLatitude(), screenTopRight.getLongitude(), 0);
-                    setProgressBarVisibility(true);
-                    savePlacesAsKML(screenTopRightLatLng,screenBottomLeftLatLng);
+                    progressDialog.setTitle("Saving KML File");
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:417]: Would you mind internationalizing this string too?

File: app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java [hunk 2]
```diff
@@ -377,14 +384,17 @@ public boolean onMenuItemClick(MenuItem item) {
...
+                        .fromPixels(mapView.getWidth(), 0);
+                    IGeoPoint screenBottomLeft = mapView.getProjection()
+                        .fromPixels(0, mapView.getHeight());
                     fr.free.nrw.commons.location.LatLng screenTopRightLatLng = new fr.free.nrw.commons.location.LatLng(
                         screenBottomLeft.getLatitude(), screenBottomLeft.getLongitude(), 0);
                     fr.free.nrw.commons.location.LatLng screenBottomLeftLatLng = new fr.free.nrw.commons.location.LatLng(
                         screenTopRight.getLatitude(), screenTopRight.getLongitude(), 0);
-                    setProgressBarVisibility(true);
-                    savePlacesAsGPX(screenTopRightLatLng,screenBottomLeftLatLng);
+                    progressDialog.setTitle("Saving GPX File");
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java:395]: Would you mind internationalizing this string too?

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
Great improvement!
Just a minor correction for internationalization needed and I will merge.

**Comment by kanahia1** [comment]
@nicolas-raoul, Sorry for the delay. I have fixed issue in the newer commit https://github.com/commons-app/apps-android-commons/pull/5654/commits/3c864d826b662a8e4500e18e1502058a3a075d06

**Review by nicolas-raoul — APPROVED** [review]

**Pull request merged**